test/constants.ts

```diff
@@ -79,3 +79,28 @@ export const buildItem = (): Item => {
     },
   };
 };
+
+export const GRAASP_ACTOR: Actor = {
+  id: 'actorid',
+};
+
+export const CATEGORY_IDS_LIST = [
```

Comment:
Maybe add some comments what's the meaning of this variable: why it's a 2-length array, the comma, etc.